src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -299,27 +299,27 @@ void EnDoor_Update(Actor* thisx, GlobalContext* globalCtx) {
 s32 EnDoor_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
     s32 pad;
     TransitionActorEntry* transitionEntry;
-    Gfx** temp_a2;
+    Gfx** doorDLists;
     s32 pad2;
-    s16 phi_v0_2;
-    s32 phi_v0;
+    s16 rotDiff;
+    s32 doorDListIndex;
     EnDoor* this = (EnDoor*)thisx;
 
     if (limbIndex == 4) {
-        temp_a2 = D_809FCEE4[this->dListIndex];
+        doorDLists = sDoorDLists[this->dListIndex];
         transitionEntry = &globalCtx->transiActorCtx.list[(u16)this->actor.params >> 0xA];
         rot->z += this->actor.world.rot.y;
         if ((globalCtx->roomCtx.prevRoom.num >= 0) ||
             (transitionEntry->sides[0].room == transitionEntry->sides[1].room)) {
-            phi_v0_2 = ((this->actor.shape.rot.y + this->skelAnime.jointTable[3].z) + rot->z) -
-                       Math_Vec3f_Yaw(&globalCtx->view.eye, &this->actor.world.pos);
-            *dList = (ABS(phi_v0_2) < 0x4000) ? temp_a2[0] : temp_a2[1];
+            rotDiff = ((this->actor.shape.rot.y + this->skelAnime.jointTable[3].z) + rot->z) -
+                      Math_Vec3f_Yaw(&globalCtx->view.eye, &this->actor.world.pos);
+            *dList = (ABS(rotDiff) < 0x4000) ? doorDLists[0] : doorDLists[1];
```

Comment:
how come you have yaw diff above in a different place and rotDiff here?
I guess a better question is what is this code doing

I would assume that rot.z from the skeleton stuff is actually the yaw, but is using z cause model space stuff. But that is my assumption without actually investigating

Reply:
I have no idea what this code is doing. I assume pick a "door side" but it's quite cryptic
I went with more generic rotDiff because I didn't actually look at the skeleton to check if the 3rd bone z rotation is actually a yaw
I'll take a look at improving `gDoorSkel` doc in this pr

Reply:
Are we waiting for more investigation on this?
`rotDiff` seems fine to me as a generic name either way, fwiw

Reply:
yea i was waiting for a followup, but yea there will prob be more docs passes on this stuff later anyway. so its fine if not

Reply:
I didn't forget but there's a lack of tool capabilities to investigate this efficiently

I can revert this one variable if you want, but this code needs more than a named variable (e.g. comments) to be legible anyway

Reply:
nah its more than fine